test/Passes/DefaultPipeline/tpp-conversion.mlir

```diff
@@ -63,6 +63,7 @@ func.func @linalg_dialect_expect_fused_brgemm(
 // CHECK-LABEL: linalg_dialect_expect_fused_brgemm
 // CHECK-SAME:  %[[ARG0:.+]]: tensor<1x3x2xf32>, %[[ARG1:.+]]: tensor<1x2x3xf32>, 
 // CHECK-SAME:  %[[ARG2:.+]]: tensor<3x3xf32>, %[[ARG3:.+]]: tensor<1x3xf32>
-// CHECK: %{{.+}} = tpp.fused_brgemm [unary = relu, binary = add] 
-// CHECK-SAME:  (%[[ARG0]] : tensor<1x3x2xf32>, %[[ARG1]] : tensor<1x2x3xf32>, %[[ARG2]] : tensor<3x3xf32>, %[[ARG3]] : tensor<1x3xf32>) -> (tensor<3x3xf32>)
-
+// CHECK: %{{.+}} = tpp.brgemm
```

Comment:
Nope, still needs to match a fused_brgemm, until we remove the TPP dialect. Let's not revert behaviour, even if we'll remove it soon.

Reply:
reverted

Reply:
test still shows `brgemm`, not fused. did you push the commit that reverts it?